modules/cnf-measuring-latency.adoc

```diff
@@ -37,6 +37,11 @@ The tests introduce the following environment variables:
 |`LATENCY_TEST_RUNTIME`
 |Specifies the amount of time in seconds that the latency test must run. The default value is 300 seconds.
 
+[NOTE]
+====
+To prevent the Ginkgo test suite from timing out before the latency tests complete, set the `-ginkgo.timeout` flag to a value greater than the `LATENCY_TEST_RUNTIME` value. The default timeout value for the Ginkgo test suite is 1 hour.
```

Comment:
Thank you @rohennes  for this adjustment. 
let's please be clear that the timeout behavior is because the tests us ginkgo v2. 
another important point that was discussed and agreed offline with @swatisehgal was that LATENCY_TEST_DELAY (if set; it is 0 by default) should also be taken into consideration when setting the timeout. so the ginkgo timeout should be greater than LATENCY_TEST_RUNTIME and LATENCY_TEST_DELAY + 2 mins.

Reply:
Updated, let me know if I captured this correctly.

Also, for all the commands in the latency section, I think we should add the timeout flag by default? Is this the below example the correct syntax/placement for the flag? (I tried to test locally but got the error "Error: SELinux relabeling of /home/rohennes is not allowed")

`podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e FEATURES=performance -e ROLE_WORKER_CNF=worker-cnf -e LATENCY_TEST_CPUS=10 -e LATENCY_TEST_RUNTIME=600 -e MAXIMUM_LATENCY=20 registry.redhat.io/openshift4/cnf-tests-rhel8:v4.14 /usr/bin/test-run.sh -ginkgo.v -ginkgo.timeout="2h" -ginkgo.focus="oslat"`

Reply:
> I can imagine a user copy/pasting one of the podman commands, hitting a timeout error, then trying to understand what went wrong. So I think adding the timeout flag to the podman run command in the test examples might be good too.


I strongly agree

Reply:
Yes, please go ahead add update the examples on the page you just mentioned too. Thanks!

Reply:
Thanks. I updated accordingly. Let me know if the flag is in the correct position, with correct syntax